readthedocs/restapi/views/model_views.py

```diff
@@ -117,6 +117,13 @@ def valid_versions(self, request, **kwargs):
             'flat': version_strings,
         })
 
+    @detail_route(permission_classes=[permissions.IsAdminUser])
+    def environment_variables(self, request, *_, **__):
+        return Response({
+            env.name: env.value
+            for env in self.get_object().environmentvariable_set.all()
+        })
```

Comment:
Why make this a separate route? We already have a admin-only serializer for Project that is used for fields only used by our builders like this. I'd rather it go there and save the extra request to the API.

Reply:
Good point!